infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationService.kt

```diff
@@ -778,6 +778,137 @@ class PublicationService @Autowired constructor(
         return asCsvFile(orderedPublishedItems, timeZone ?: ZoneId.of("UTC"))
     }
 
+    private fun diffTrackNumber(newTrackNumber: TrackLayoutTrackNumber, oldTrackNumber: TrackLayoutTrackNumber?) =
```

Comment:
Pihviä tästä eteenpäin

Reply:
Nämä diffaus funkkarit ovat näemmä selkesti omaa settiään ja puhtaita funkkareita -> voisi heittää omaan tiedostoonsa, vaikka PublicationDiff.kt tms. Sieltä ne on myös tarvittaessa helpompi testata kun ei tarvitse servicen kontekstia alustaa (..Test eikä ..IT)

Reply:
Joo, näiden paikkaa ei ole vielä mietitty juuri ollenkaan muuten kuin että tarvivat servicejä -> toistaiseksi täällä. Sitten kun saan tämän koko höskän ylipäätään fiksusti toimivaan muottiin, niin tarkoitus olisi käydä koko koodi vielä läpi ja siirrellä ja pilkkoa se